src/components/Search/search.css

```diff
@@ -0,0 +1,19 @@
+.display {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 20px;
+}
+
+.btn-secondary,
+.btn-secondary:focus,
+.btn-secondary:active {
+  color: #fff;
+  background-color: #00838f !important;
```

Comment:
We should avoid using `!important` wherever possible! It can make it difficult to reason about CSS rules: https://www.w3schools.com/css/css_important.asp